hedera-node/hedera-schedule-service-impl/src/main/java/com/hedera/node/app/service/schedule/impl/handlers/HandlerUtility.java

```diff
@@ -169,6 +169,7 @@ static HederaFunctionality functionalityForType(final DataOneOfType transactionT
             case NODE_CREATE -> HederaFunctionality.NODE_CREATE;
             case NODE_UPDATE -> HederaFunctionality.NODE_UPDATE;
             case NODE_DELETE -> HederaFunctionality.NODE_DELETE;
+            case TOKEN_CANCEL_AIRDROP -> HederaFunctionality.TOKEN_CANCEL_AIRDROP;
```

Comment:
Does this need other `TOKEN_CLAIM_AIRDROP` and others also ?

Reply:
Yes! Apparently, we've missed adding the airdrop and claim bodies in `schedulable_transaction_body.proto`. I've added them in this PR https://github.com/hashgraph/hedera-protobufs/pull/391/files#diff-e6cc8bceeac474ff81e068d645f139c701060ac98dba6fdcb94c986228c6b11c and added the new mapping in the file. Good catch!